lib/patch-series.ts

```diff
@@ -858,14 +857,10 @@ export class PatchSeries {
     }
 
     protected async publishBranch(tagName: string): Promise<void> {
-        if (!this.project.publishToRemote || this.options.dryRun) {
-            return;
-        }
-
```

Comment:
In the command-line version, back when it was still a Unix shell script, and then a Javascript script, I used to use `--dry-run` a lot, and I really did not want anything to be published in that case.

So I would like to keep this here.

If all of the calls already have this guard, I would rather unguard the calls and keep the guard in the function, as that makes it much safer to use it in the future.